src/client/Client.cc

```diff
@@ -204,6 +204,21 @@ int Client::CommandHook::call(
 
 // -------------
 
+int Client::get_fd_inode(int fd, InodeRef *in) {
+  int r = 0;
+  if (fd == AT_FDCWD) {
```

Comment:
Expose this as `CEPHFS_AT_FDCWD` in `src/include/fs_types.h`. Copy the Linux value.